app/jobs/export/export_base_job.rb

```diff
@@ -11,7 +11,7 @@ class Export::ExportBaseJob < BaseJob
 
   attr_reader :exporter
 
-  def initialize(format, user_id, options = {})
+  def initialize(format, user_id, options: {})
```

Comment:
Wenn du das wirklich zu einem Keyword Argument ändern willst, musst du auch in allen Wagons noch nachschauen ob damit etwas kaputt geht. Ich sehe ehrlich gesagt den Grund noch nicht ganz, warum wir das ändern. Aber bei einer kurzen Suche habe ich [zwei](https://github.com/hitobito/hitobito_sbv/blob/master/app/jobs/export/song_counts_export_job.rb) [Exports](https://github.com/hitobito/hitobito_sbv/blob/master/app/jobs/export/group_participations_export_job.rb) beim SBV gefunden die du anpassen musst. Und theoretisch ist das ein breaking change für hitobito-Wagons die nicht wir entwickeln.

Reply:
guete Punkt !
würdest du denn die optionen mit den group role ids eher mit einem getter setzen anstatt im constrüctör? oder hast du eine andere Idee wie wir diese optionen reinkriegen?